src/lib/SVGMap/SVGMap.svelte

```diff
@@ -0,0 +1,190 @@
+<script>
+  import { reducedMotion } from "../stores";
+  import { readable, writable } from "svelte/store";
+  import { setContext, onMount } from "svelte";
+  import { geoAlbersUsa } from "d3-geo";
+  import { zoom, zoomIdentity } from "d3-zoom";
+  import { select } from "d3-selection";
+  import ZoomControls from "./ZoomControls.svelte";
+
+  /*
+   * An array of geojson features to be displayed on the map. The map will scale the projection to fit this set of features.
+   */
+  export let features;
+  export let projection = geoAlbersUsa;
+  export let zoomable = false;
+
+  /*
+   * Where the zoom control UI should be positioned.
+   * @type {"bottom-left" | "bottom-right" | "top-left" | "top-right"}
+   */
+  export let controlPosition = "bottom-right";
+  export let height = 800;
+
+  /*
+   * Optional aria role string to be applied to SVG container. By default, the SVG is hidden from the accessiblity tree. If you add an ariaRole here, any layers should also be given an ariaRole.
+   * @type { string } [ariaRole = undefined]
+   */
+  export let ariaRole = undefined;
+
+  /*
+   * Optional aria label string to be applied to SVG container. By default, the SVG is hidden from the accessiblity tree and should include a descriptive label. If you add an ariaRole this property can be left undefined;
+   * @type { string } [ariaRole = undefined]
+   */
+  export let ariaLabel = undefined;
+
+  /**
+   * Whether or not the map should zoom when scroll wheel is used on map.
+   * @type {"no" | "yes" | "ctrl"}
+   */
+  export let scrollWheel = "no";
+
+  // create stores of map global settings to add to context
+  $: featuresStore = readable(features);
+  let width = 500;
+
+  // size to fit projection to
+  $: fitSizeRange = [width, height];
+
+  // setup the scaled projection function
+  $: projectionFn = projection().fitSize(fitSizeRange, {
+    type: "FeatureCollection",
+    features: features
+  });
+
+  // set up a store to hold the scaled projection function
+  const projectionStore = writable(projectionFn);
+  // update the store any time the projection function updates
+  $: projectionStore.set(projectionFn);
+
+  // initialize a transform store in case zoom is turned on
+  // this will be where we store the output of the d3-zoom behaviour and broadcast it
+  let transformStore = writable(zoomIdentity);
```

Comment:
Not sure how with this setup, but a nice to have would be setting CSS cursor property to "grab"/"grabbing" before/during drag.

Reply:
adding this as an idea to the backlog!